README.md

```diff
@@ -36,7 +36,17 @@ addSbtPlugin("com.softwaremill.clippy" % "plugin-sbt" % "0.4.1")
 
 Upon first use, the plugin will download the advice dataset from `https://scala-clippy.org` and store it in the
 `$HOME/.clippy` directory. The dataset will be updated at most once a day, in the background. You can customize the
-dataset URL and local store by using the `-P:clippy:url=` and `-P:clippy:store=` compiler options.
+dataset URL and local store by setting the `clippyUrl` and `clippyLocalStoreDir` sbt options to non-`None`-values.
```

Comment:
Yes, still available. This whole section is about using Clippy from SBT, so I kept only SBT options there. There's some info on compiler plugin options in `Alternative ways to use Clippy`